gitpod-io/gitpod
# Use the actual billing cycle dates in all usage-based Billing pages

## Description


Use the actual billing cycle dates in all usage-based Billing pages

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/14363

## How to test


1. Create a team called "Gitpod [Something]"
  a. this enables usage-based for your user
1. Go to your personal billing page (`/billing`)
  a. notice the billing cycle dates (from ~now to ~a month from now)
1. Upgrade by adding a credit card
  a. notice the new billing cycle dates (from ~now to the last UTC millisecond of the month)

**Hint**: You can hover on the billing cycle dates to see their exact times.

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`
